src/components/dashboard/Dashboard.js

```diff
@@ -570,9 +578,15 @@ const Dashboard = props => {
       const minScrollRequiredISH = headerLarge ? minScrollRequired : minScrollRequired * 2
       const scrollPositionISH = headerLarge ? scrollPosition : scrollPosition + minScrollRequired
       if (feeds && feeds.length && feeds.length > 10 && scrollPositionISH > minScrollRequiredISH) {
-        headerLarge && setHeaderLarge(false)
+        if (headerLarge) {
+          setHeaderLarge(false)
+          setGetNextFeedAllowed(false)
+        }
       } else {
-        !headerLarge && setHeaderLarge(true)
+        if (headerLarge) {
```

Comment:
there was `!headerLarge` check before
why it's inverted now ?

Reply:
it's a mistake. I reverted it back